features/user_can_login_and_logout.feature

```diff
@@ -0,0 +1,39 @@
+Feature: User has access to login and logout
```

Comment:
Should we clarify why this feature is being implemented? For example: To save and update user information the user can access their account with login and logout.

Reply:
Yeah this can be renamed, will activate my 'Shakespeare' mode now and come up with something better :D

Reply:
Now discussing it again with Zuzanna, I think the Feature title is fine, I think Thomas was just referring to the story for the feature has to define the purpose, regarding the reasoning for the implementation.